cli/tests/services.bats

```diff
@@ -1518,8 +1519,36 @@ EOF
 
   # Make sure services have stopped
   timeout 1s bash -c '
-    while "$FLOX_BIN" services status; do
+    both_services_stopped() {
+      one_status="$("$FLOX_BIN" services status --json one | jq -r -c .status)"
+      two_status="$("$FLOX_BIN" services status --json two | jq -r -c .status)"
+      [ "$one_status" = "Stopped" ] && [ "$two_status" = "Stopped" ]
+    }
+    
+    while ! both_services_stopped; do
       sleep .1
     done
   '
 }
+
+@test "status: prints requested services (before start)" {
+  setup_sleeping_services
+
+  run "$FLOX_BIN" services status --json one
+  assert_success
+
+  statuses="$(echo "$output" | jq --slurp -c 'map(.status)')"
+  assert_equal "$statuses" '["Stopped"]'
+}
+
+@test "status: prints requested services (after start)" {
+  setup_sleeping_services
+
+  run "$FLOX_BIN" activate -s -- bash -c '
+    "$FLOX_BIN" services status --json one
+  '
+  assert_success
+
+  statuses="$(echo "$output" | jq --slurp -c 'map(.status)')"
+  assert_equal "$statuses" '["Running"]'
+}
```

Comment:
This duplicates an existing test (which incidentally doesn't need to use `cat` and has the wrong indentation): https://github.com/flox/flox/blob/cc2ac7c4efaee93c66a0c2c29d02bdf49ff898a4/cli/tests/services.bats#L891-L902

Reply:
This actually doesn't cover the same thing. The existing test covers that all services are listed without a request for a specific service. The new test covers that only the requested service is listed. I added both of these tests because nothing (except the `wait_for_service_status` helper in one test) broke. I'll move them closer to the existing test, which I also didn't know existed lol